partner_view_adjust/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module does the following:
+
+- Add to the ref fields to the partner list view.
```

Comment:
This module makes adjustments to partner views.

The module should not include changes to models or business logic.